zeebe/atomix/cluster/src/main/java/io/atomix/raft/roles/PassiveRole.java

```diff
@@ -106,7 +106,11 @@ public CompletableFuture<Void> stop() {
   /** Truncates uncommitted entries from the log. */
   private void truncateUncommittedEntries() {
     if (role() == RaftServer.Role.PASSIVE && raft.getLog().getLastIndex() > raft.getCommitIndex()) {
-      raft.getLog().deleteAfter(raft.getCommitIndex());
+      try {
+        raft.getLog().deleteAfter(raft.getCommitIndex());
+      } catch (final FlushException e) {
+        throw new RuntimeException(e);
```

Comment:
:wrench: Not a big fan of throwing generic runtime exceptions. If we're just rethrowing, why not sneaky throw via `LangUtil.rethrowUnchecked`? 

:x: But also, why are we rethrowing here? From what I can see, this will likely cause us to go inactive. What we would want is likely to just retry forever, no? Or maybe with a set number of time, before going inactive. Otherwise one blip and you end up inactive, I think. Note that I'm not 100% sure, so I could be wrong - but looking at the code, it looks like that's what will happen.